src/components/gui/gui.jsx

```diff
@@ -35,16 +35,19 @@ const messages = defineMessages({
 
 const GUIComponent = props => {
     const {
+        activeTabIndex,
         basePath,
+        blocksTabVisible,
         children,
+        costumesTabVisible,
         enableExtensions,
         intl,
         feedbackFormVisible,
         vm,
         previewInfoVisible,
+        soundsTabVisible,
         onExtensionButtonClick,
-        onTabSelect,
-        tabIndex,
+        onActivateTab,
```

Comment:
I know we removed the sort-prop-types lint rule, but I think this could still have a better order. How about moving all the `...Visible` props to just before the `on...` handler props, with the remaining props alphabetical? Alternatively let's fix the alphabetization (`intl` and `vm` are incorrectly sorted)